exporter/elasticsearchexporter/README.md

```diff
@@ -197,7 +197,8 @@ The behaviour of this bulk indexing can be configured with the following setting
   - `interval` (default=30s): Write buffer flush time limit.
 - `retry`: Elasticsearch bulk request retry settings
   - `enabled` (default=true): Enable/Disable request retry on error. Failed requests are retried with exponential backoff.
-  - `max_requests` (default=3): Number of HTTP request retries.
+  - `max_requests` (DEPRECATED, use retry::max_retries instead): Number of HTTP request retries including the initial attempt.
+  - `max_retries` (default=3): Number of HTTP request retries. To disable retries, set `retry::enabled` to `false` instead of setting `max_retries` to `0`.
```

Comment:
Does it matter that the default value is now technically one more retry than before? (Just want to call out potential change in behavior of the default configuration for users here)

Reply:
Oh! nice catch. Yes let me fix that.